litestar/_asgi/routing_trie/mapping.py

```diff
@@ -143,8 +143,7 @@ def configure_node(
         node.path_parameters = {}
 
     if isinstance(route, HTTPRoute):
-        for method, handler_mapping in route.route_handler_map.items():
-            handler, _ = handler_mapping
+        for method, handler in route.route_handler_map.items():
```

Comment:
I think we can get rid of the references to the handlers within the asgi router entirely, and have the resolution back to a handler happen at a later point via the routes. I'm gonna leave this as-is for now though and maybe later handle this change in a separate PR